### PR TITLE
Fix Issue with emails not having email headers

### PR DESCRIPTION
**Pull request by cameronblaikie**
## Description

When you reply/forward an email the email header information like "From, Date, To, CC, Subject" did not display. Now the Header displays all relative information in the Email header when Replying/Forwarding.

## Motivation and Context

Now all information is displayed in the email header when in compose for replying/forwarding

## How To Test This

Reply / Reply To All / Foward any email and see that there is a "From, Date, To, CC, Subject".

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Comment by pgorod** [comment]
I confess I don't understand why we're putting `<br>` HTML tags into the `description` field, not the `description_html` field... although that seems to be happening in that subroutine even _before_ these proposed changes...

**Review by jack7anderson7 — APPROVED** [review]

File: modules/Emails/Email.php [hunk 1]
```diff
@@ -1929,6 +1929,30 @@ public function getForwardHeader()
...
+    public function getEmailHeaderOutgoing($emailCc)
+    {
+        global $mod_strings;
+        global $current_user;
+
+        $mailAccount = BeanFactory::getBean('InboundEmail', $this->mailbox_id);
+
+        $toEmail = $mailAccount->email_user;
+
+        //$from = str_replace(array("&gt;","&lt;"), array(")","("), $this->from_name);
```
Comment (clemente-raposo) [inline modules/Emails/Email.php:1941]: Hi @cameronblaikie, thank you for your PR, can you remove the commented code please?

File: modules/Emails/Email.php [hunk 2]
```diff
@@ -1929,6 +1929,30 @@ public function getForwardHeader()
...
+        $ret = "<br /><br />";
+        $ret .= $this->replyDelimiter . "{$mod_strings['LBL_FROM']} {$this->reply_to_addr}<br />";
+        $ret .= $this->replyDelimiter . "{$mod_strings['LBL_DATE_SENT_RECEIVED']} {$this->date_sent_received}<br />";
+        $ret .= $this->replyDelimiter . "{$mod_strings['LBL_TO']} {$toEmail}<br />";
+        $ret .= $this->replyDelimiter . "{$mod_strings['LBL_CC']} {$emailCc}<br />";
+        $ret .= $this->replyDelimiter . "{$mod_strings['LBL_SUBJECT']} {$subject}<br />";
+        $ret .= $this->replyDelimiter . "<br />";
+
+        return $ret;
+        //return from_html($ret);
```
Comment (clemente-raposo) [inline modules/Emails/Email.php:1953]: Hi @cameronblaikie, same as https://github.com/salesagility/SuiteCRM/pull/9575/files#r1142483186

**Comment by serhiisamko091184** [comment]
Hello all!
@cameronblaikie 
would you be so kind to have a look at these comments also?
Thanks in advance!

Regards,
Serhii

**Comment by serhiisamko091184** [comment]
Hello, @cameronblaikie 

The PR has been marked as stale because there has been no recent activity. It will be closed if no further activity occurs.
Thanks for your contributions.

Regards,
Serhii

**Comment by serhiisamko091184** [comment]
Hey @cameronblaikie 

Closing this PR due to inactivity. 
Thank you.

Regards,
Serhii

**Pull request closed without merging**